packages/code-studio/src/settings/SettingsMenu.tsx

```diff
@@ -84,11 +83,28 @@ export class SettingsMenu extends Component<
     onDone();
   }
 
+  handleExportSupportLogs(): void {
+    const { serverConfigValues } = this.props;
+    exportLogs(undefined, Object.fromEntries(serverConfigValues));
+  }
+
   render(): ReactElement {
-    const version = import.meta.env.npm_package_version;
+    const uiVersion = import.meta.env.npm_package_version;
     const supportLink = import.meta.env.VITE_SUPPORT_LINK;
     const docsLink = import.meta.env.VITE_DOCS_LINK;
 
+    const { serverConfigValues } = this.props;
+    const barrageVersion = serverConfigValues.get('barrage.version');
+    const javaVersion = serverConfigValues.get('java.version');
+    const deephavenVersion = serverConfigValues.get('deephaven.version');
+
+    const getRow = (text: string, ver?: string) => (
+      <div className="detailed-server-config-row">
+        <div>{text}</div>
+        <div>{ver}</div>
+      </div>
```

Comment:
```suggestion
      <>
        <div>{text}</div>
        <div>{ver}</div>
      </>
```

If you use a css grid, you don't need this extra wrapper.